template/python/function/handler.py

```diff
@@ -0,0 +1,11 @@
+from pydantic import BaseModel
+from fastapi import Request
+from uuid import uuid4, UUID
+
+
+class RandomUUID(BaseModel):
+    uuid: str
+
+
+def handler(request: Request) -> RandomUUID:
+    return RandomUUID(uuid=str(uuid4()))
```

Comment:
Why the function `requirements.txt` is empty ? When the user will develop the code locally (outside the template) potentially there is not the requirements already available ?

Reply:
added faker for example